authgear/templates/authgear-main-server.yaml

```diff
@@ -165,6 +165,8 @@ spec:
           value: {{ .Values.authgear.appNamespace | quote }}
         - name: CUSTOM_RESOURCE_DIRECTORY
           value: {{ .Values.authgear.appCustomResources.path | quote }}
+        - name: SHARED_AUTHGEAR_ENDPOINT
+          value: {{ .Values.portalServer.authgear.endpoint }}
```

Comment:
It should be `.Values.authgear.portalServer.authgear.endpoint`

---

How I found this out

```
$ helm template authgear
Error: template: authgear/templates/authgear-main-server.yaml:169:27: executing "authgear/templates/authgear-main-server.yaml" at <.Values.portalServer.authgear.endpoint>: nil pointer evaluating interface {}.authgear

Use --debug flag to render out invalid YAML
```